### DISCUSSION TITLE
GRPC over HTTP/2 in Deno

### DISCUSSION THREAD

**Discussion by caleblloyd** [connectrpc/connect-es#1184] [Show and tell]
As of Deno 1.46.1, `connect-es` using GRPC over HTTP/2 works in Deno!

Here is a demo: https://github.com/caleblloyd/connect-es-deno-demo

I tested using `connect-es` version 2.x

**Comment by reyawn** [comment]
I've found that hosting a `connect-es` server does not work, though.
